spec/lib/services/profiles-api-service-spec.js

```diff
@@ -84,6 +84,20 @@ describe("Http.Services.Api.Profiles", function () {
             }
         };
 
+        var profileReq = {
+            query: {
+                'ips': ['ip1', 'ip2'],
```

Comment:
Suggest to use valid IP/Mac even in unit-test, we my add some validation logic in future to screen out invalid IP/Mac address.